deploy/registration-service.yaml

```diff
@@ -104,26 +112,67 @@ objects:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
-                - name: REGISTRATION_ENVIRONMENT
+                - name: REGISTRATION_SERVICE_CONFIG_MAP_NAME
+                  value: ${CONFIG_MAP_NAME}
+                - name: REGISTRATION_SERVICE_SECRET_NAME
+                  value: ${SECRET_NAME}
+                - name: WATCH_NAMESPACE
+                  value: ${NAMESPACE}
+                - name: REGISTRATION_AUTH_CLIENT_LIBRARY_URL
                   valueFrom:
                     configMapKeyRef:
                       name: registration-service
-                      key: environment
-                - name: REGISTRATION_AUTH_CLIENT_LIBRARY_URL
+                      key: auth_client.library_url
+                - name: REGISTRATION_AUTH_CLIENT_PUBLIC_KEYS_URL
                   valueFrom:
                     configMapKeyRef:
                       name: registration-service
-                      key: auth_client.library_url
-                - name: REGISTRATION_AUTH_CLIENT_CONFIG_RAW
+                      key: auth_client.public_keys_url
+                - name: REGISTRATION_AUTH_CLIENT_CONFIG_RAW_REALM
```

Comment:
Why did you replace the `REGISTRATION_AUTH_CLIENT_CONFIG_RAW` with multiple entries? this is not how it is being configured

Reply:
i changed how it was being configured. as mentioned in another comment above, it seemed a bit inconsistent to have a blob of configurations. having it as separate fields also makes it easier (imo) when creating a configmap. lastly if we ever needed any of these fields we would need to decode it in order to retrieve it (see tests).. but in any case, i've changed it back..